openstef/pipeline/create_component_forecast.py

```diff
@@ -53,12 +55,20 @@ def create_input(
     input_df["hour"] = input_df.index.hour
     input_df["minute"] = input_df.index.minute
 
-    input_df["var0"] = input_df["total_substation"].var()
+    input_df["var0"] = input_df["total_load"].var()
     input_df["var1"] = input_df["radiation"].var()
     input_df["var2"] = input_df["windspeed_100m"].var()
 
-    input_df["sem0"] = input_df["total_substation"].sem()
+    input_df["sem0"] = input_df["total_load"].sem()
     input_df["sem1"] = input_df["radiation"].sem()
+    input_df["sem2"] = input_df["windspeed_100m"].sem()
+
+    # Features for the new model
+    # Periodic Month feature
+    c = (1 / 11) * np.pi - (1 / 365)
+    n = pd.to_datetime(input_df.index).strftime("%m").tolist()
+    n = np.array([float(x) for x in n])
+    input_df["month_ff"] = np.sin(c * (n - 1))
```

Comment:
What prevents you from using n directly from `strftime` (without `.tolist()`) as the input for the sine function?